packages/server-core/src/media/oembed/oembed.class.ts

```diff
@@ -47,13 +48,13 @@ export class OembedService implements ServiceInterface<OembedType | BadRequest |
     if (!queryURL) return new BadRequest('Must provide a valid URL for OEmbed')
 
     const url = new URL(queryURL)
-    const serverSettingsResult = (await this.app.service(serverSettingPath).find()) as Paginated<ServerSettingType>
+    const serverSettingsResult = (await this.app.service(engineSettingPath).find()) as Paginated<EngineSettingType>
```

Comment:
Minor thing but couldn't we just query for engineSettings where category is 'server'?